pkg/proxy/backup.go

```diff
@@ -18,7 +25,40 @@ func (p *Proxy) SnapshotBackup(ctx context.Context, req *rpc.EngineSnapshotBacku
 		IsIncremental: false,
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.